odl/solvers/advanced/proximal_operators.py

```diff
@@ -846,6 +898,61 @@ def proximal_cconj_l1(space, lam=1, g=None, isotropic=False):
     prox_factory : function
         Factory for the proximal operator to be initialized
 
+    Notes
+    -----
+    The :math:`L_1`-norm :math:`F` is the functional
+
+        .. math::
+
+            F(x) = \\lambda ||x - g||_1.
+
+    The convex conjugate :math:`F^*` of :math:`F` is given by the indicator
+    function of the set :math:`box(\\lambda)`
+
+        .. math::
+
+            F^*(y) = \\lambda I_{box(\\lambda)} \left(
+            \left| \\frac{y}{\\lambda} \\right| +
+            \left\\langle \\frac{y}{\\lambda}, g \\right\\rangle
+            \\right)
```

Comment:
> I will first give the same answer as I did somewhere else: I have simply typeset this

Sure :-)



> But I don't think it is necessarily that easy to simplify the argument of the indicator function... you can maybe try some completion of squares, but that I don't think it is very useful.

I think it's quite trivial. You can extract a 1/lam from both summands.

My suspicion is that the formula is simply wrong since the scaling of the argument by `1/lam` is the same as scaling the box by `lam`. Probably it should be either 
```
box(1)(|y / lam| + <y / lam, g>)
```
or
```
box(lam)(|y| + <y, g>)
```